ui/selectors/institutional/selectors.js

```diff
@@ -52,27 +52,42 @@ export function getCustodianIconForAddress(state, address) {
 }
 
 export function getIsCustodianSupportedChain(state) {
-  const selectedIdentity = getSelectedIdentity(state);
-  const accountType = getAccountType(state);
-  const providerConfig = getProviderConfig(state);
-
-  const supportedChains =
-    accountType === 'custody'
-      ? getCustodyAccountSupportedChains(state, selectedIdentity.address)
-      : null;
-
-  return supportedChains?.supportedChains
-    ? supportedChains.supportedChains.includes(
-        hexToDecimal(providerConfig.chainId),
-      )
-    : true;
+  try {
+    const selectedIdentity = getSelectedIdentity(state);
+    const accountType = getAccountType(state);
+    const providerConfig = getProviderConfig(state);
+
+    if (!selectedIdentity || !accountType || !providerConfig) {
+      throw new Error('Invalid state');
+    }
+
+    if (accountType !== 'custody') {
```

Comment:
Optional: can we have another MR to move `custody` to an enum in the future? also can include `default`

Reply:
Yes, good one. I'll talk to the team to do it :)